article.html

```diff
@@ -15,7 +15,6 @@
             <div class="article">
                 <h1>Article header here</h1>
                 <h2>Author and publish date here</h2>
-                <hr color="black" size="10">

```

Comment:
I removed this black line that spans across the page in layout 3 as it seemed unnecessary.